### ISSUE TITLE
Unable to install, Downloading...

### ISSUE THREAD

**Issue by johnmasayume** [tailscale-dev/deck-tailscale#19]
Hi there, i having some issue here while installing using sudo bash tailscale.sh

(1)(deck@steamdeck ~)$ cd /home/deck/Downloads/deck-tailscale-main/
(deck@steamdeck deck-tailscale-main)$ sudo bash tailscale.sh
[sudo] password for deck: 
Getting version...got 1.56.1.
Downloading...   

here is my log above showing in the Konsole. The Downloading... is stuck for quite long time.

also i did not clone this repo, i just downloaded all those 4 files into zip file and extract to a folder. I cd into that folder and run sudo bash tailscale.sh

i am not sure why is not downloading after waiting for 15mins

also previously did found someone online github have a flatpak for tailscale on steamdeck, i might installed that as well that causes this installation to fail. but i am not sure how to remove them
https://github.com/Str-iker/steamdeck-tailscale

**Comment by legowerewolf** [comment]
What happens if you try this command?
```
curl -s "https://pkgs.tailscale.com/stable/tailscale_1.56.1_amd64.tgz" -o tailscale.tgz
```

**Comment by johnmasayume** [comment]
> What happens if you try this command?
> 
> ```
> curl -s "https://pkgs.tailscale.com/stable/tailscale_1.56.1_amd64.tgz" -o tailscale.tgz
> ```

i think it manages to download it? my output is something like this
(1)(deck@steamdeck ~)$ curl -s "https://pkgs.tailscale.com/stable/tailscale_1.56.1_amd64.tgz" -o tailscale.tgz
 
(deck@steamdeck ~)$ 
(deck@steamdeck ~)$ 

but where is the downloaded file located and what should i do next after this?

**Comment by legowerewolf** [comment]
alright, so the install script ran? I'm not sure if the new method updates your PATH; @diericx can you help?

**Comment by diericx** [comment]
I've had the download hang before as well. I thought I was getting rate limited but maybe not.

If truly nothing shows up it communicates to me that the binary is found but maybe the command is failing. If the binary was not in the path you would get `command not found` error message. If you have two installations it may be that the daemons are colliding.

Can you run the following commands and provide the output? I will provide my intentions behind each one for context. 

1. Find path of the `tailscale` tool
```
which tailscale
```

2. Check the status of the tailscale daemon (will not work if the command above failed)
```
tailscale status
```

3. Check tailscaled systemd service for any errors
```
sudo systemctl status tailscaled
```

**Comment by sweetbbak** [comment]
I had this exact same issue. I tried `curl -s` like it is in the script and it hung for a long time. I tried `wget` and got it to work well. So I then used `set -x` so I could track what was happening and ran `watch -n.1 -- du "${tmp}/*.tgz"` and I realized that it was downloading comically slow for some reason. I tested it and it seems inconsistent in regards to speed.

perhaps leaving out the --silent flag on curl, or using wget for the progress bar, could be helpful in letting users know that something is indeed happening or whether it is hanging up.
